### PR TITLE
#4904 add mailing list creation in the SIG/wg lifecycle document

### LINKED ISSUE #4904: Mailing list creation in the SIG/wg lifecycle document is missing important information
Placeholder bug until I can get to it, or feel free to submit a fix! 

https://github.com/kubernetes/community/blob/master/sig-wg-lifecycle.md

Doesn't mention that we need new lists to have contributors@kubernetes.io, etc, this is important! We should add a link to people to check out the mailing list guidelines just like we do for zoom, slack, etc. to point to this page:

https://github.com/kubernetes/community/blob/master/communication/mailing-list-guidelines.md

/help-wanted

### PR DESCRIPTION
**Pull request by jewar**
Fixes #4904

### CODE REVIEW

**Comment by jewar** [comment]
@parispittman @nikhita : are there any additional steps I need to do so that mailing-list-guidelines.md is a functional link? Or will this be added by the build process?
Are there maybe guidelines around links in docs?

**Comment by castrojo** [comment]
You need to add something like `[mailing-list-guidelines.md]: /communication/mailing-list-guidelines.md` at the bottom, that will make the link work.

File: sig-wg-lifecycle.md [hunk 1]
```diff
@@ -56,13 +56,13 @@ Out of scope for this document: [subproject] creation.
 ### Communicate:
 Each one of these has a linked canonical source guideline from set up to moderation and your role and responsibilities for each. We are all responsible for enforcing our [code of conduct].
 - [ ] Read [moderation.md] and understand your role in keeping our community safe
-- [ ] Create your mailing lists - One for your members and another for your chairs/leads
-- Either [GoogleGroups] OR [discuss.kubernetes.io]
```
Comment (jewar) [inline sig-wg-lifecycle.md:60]: removed discuss.kubernetes.io as we want to streamline mailing list creation. discussed in https://kubernetes.slack.com/archives/C1TU9EB9S/p1593624758123000 with @parispittman and @castrojo

**Comment by nikhita** [comment]
@jewar Thank you and congrats on your first PR to @kubernetes! :tada: :woman_dancing: 

/ok-to-test
/lgtm
/approve

**Pull request merged**